docs/api/product_variants.rst

```diff
@@ -0,0 +1,738 @@
++------------------+------------------------------------------------------------------------------------------------+
+| height           | The height of variant                                                                          |
++------------------+------------------------------------------------------------------------------------------------+
+| depth            | The depth of variant                                                                           |
++------------------+------------------------------------------------------------------------------------------------+
+| weight           | The weight of variant                                                                          |
++------------------+------------------------------------------------------------------------------------------------+
+
+.. note::
+
+    Read more about `Product Variants`__
+
+__ http://docs.sylius.org/en/latest/components/Product/models.html#variant
+
+Collection of Product Variants
+---------------------------------
+
+You can retrieve the full product variants list for selected product by making the following request:
+
+Definition
+..........
+
+.. code-block:: text
+
+    GET /api/v1/products/{productId}/variants/
+
++---------------------------------------+----------------+----------------------------------------------------------+
+| Parameter                             | Parameter type | Description                                              |
++=======================================+================+==========================================================+
+| Authorization                         | header         | Token received during authentication                     |
++---------------------------------------+----------------+----------------------------------------------------------+
+| productId                             | url attribute  | Id of product for which the variants should be displayed |
++---------------------------------------+----------------+----------------------------------------------------------+
+| limit                                 | query          | *(optional)* Number of items to display per page,        |
+|                                       |                | by default = 10                                          |
++---------------------------------------+----------------+----------------------------------------------------------+
+| sorting['name_of_field']['direction'] | query          | *(optional)* Field and direction of sorting,             |
+|                                       |                | by default 'desc' and 'createdAt'                        |
++---------------------------------------+----------------+----------------------------------------------------------+
+
+Example
+.......
+
+.. code-block:: bash
+
+    curl http://sylius.dev/api/v1/products/1/variants \
```

Comment:
Did you checked this curl? I am pretty sure you  missed a last `\`. Also, missing `$` at the beginning